typings/global.d.ts

```diff
@@ -33,3 +33,8 @@ declare namespace globalThis {
     // eslint-disable-next-line no-var
     var expect: ExpectWebdriverIO.Expect;
 }
+
+declare module "expect-webdriverio/lib/matchers" {
+    const matchers: ExpectWebdriverIO.Matchers<Promise<{ pass: boolean; message(): string }>, unknown>;
+    export default matchers;
```

Comment:
What is this for? Would it cause issues, if user adds testing library matchers?
https://github.com/gemini-testing/testplane/discussions/887#discussioncomment-8935903

Reply:
I can't import only available matchers from `expect-webdriverio`. This has been implemented only in the new version - https://github.com/webdriverio/expect-webdriverio/blob/v4.13.0/src/index.ts#L8, which support only esm modules. So this is hack to be able to use only matchers